app/to_fight.py

```diff
@@ -0,0 +1,5 @@
+def fight(fighter1: dict, fighter2: dict) -> int:
+    fighter1["hp"] -= fighter2["power"] - fighter1["protection"]
+    if fighter1["hp"] <= 0:
+        fighter1["hp"] = 0

```

Comment:
can you make it in one line?

Reply:
unfortunately no:  E701 multiple statements on one line (colon)